doc/tutorials/acrn_configuration_tool.rst

```diff
@@ -154,3 +272,133 @@ Here is the offline configuration tool workflow:
+
+      .. figure:: images/select_board_info.png
+         :align: center
+         :scale: 80%
+
+#. Choose a scenario from the “Scenario Setting” menu which lists all the scenarios
+   including default scenarios and user-defined scenarios for the board you selected
+   in the previous step. The scenario configuration xmls are located in
+   ``acrn-hypervisor/misc/xmls/config-xmls/[board]/``.
+
+   .. figure:: images/choose_scenario.png
+      :align: center
+      :scale: 80%
+
+#. It is also allowed to use a customized scenario xml by clicking the Import button.
+   The config app will auto direct to the new scenario xml once the import is completed.
+
+#. The configurable items will be displayed after one scenario is selected. Here is
+   the example of "SDC" scenario:
+
+   .. figure:: images/configure_scenario.png
+      :align: center
+      :scale: 80%
+
+   - You can editor those items directly in the text boxes, choose single or even multiple
+     items from the drop down list.
+
+   - Read-only items are marked as grey.
+
+   - Hover the mouse pointer over the item to display the description.
+
+#. Click **Export** button to save the scenario xml, you can rename it in the pop-up modal.
+
+   .. note:: All customized scenario xmls will be in user-defined groups which located in
+      ``acrn-hypervisor/misc/xmls/config-xmls/[board]/user_defined/``.
+
+   After the scenario is saved, the page will auto direct to the saved scenario xmls.
+   You can delete the configured scenario by click button **Export** -> **Remove**.
+   Before saving the scenario xml, the config app will validate the configurable items;
+   If there are errors, the config app will list all wrong configurable items and show errors as below:
+
+   .. figure:: images/err_acrn_configuration.png
+      :align: center
+      :scale: 80%
+
+#. Click **Generate Board SRC** to save current scenario setting and then generate board
+   source codes in ``acrn-hypervisor/hypervisor/arch/x86/configs/[board]/``.
+
+#. Click **Generate Scenario SRC** to save current scenario setting and then generate
+   scenario source codes in ``acrn-hypervisor/hypervisor/scenarios/[scenario]/``.
+
+#. **Launch Setting** is quite similar with **Scenario Setting**:
+
+   a. Upload board info or select one board as current board.
+
+   #. Import your local launch setting xml by click button **Import** or select one launch
+      setting xml from menu.
+
+   #. Select one scenario for current launch setting from the drop down box **Select Scenario**.
+
+   #. Configure the items for current launch setting.
+
+   #. Save current launch setting to user defined xml files by click the button **Export**.
+      The config app will validate current configuration and will list all wrong configurable
+      items and show errors.
+
+   #. Click the button **Generate Launch Script** to save current launch setting and then
+      generate launch script.
+
+      .. figure:: images/generate_launch_script.png
```

Comment:
the sizes of controls in the page look larger than others, suggest to screenshot them in the same scaling size